lib/furoshiki/linux_app.rb

```diff
@@ -0,0 +1,45 @@
+require 'furoshiki/base_app'
+require 'plist'
```

Comment:
I see lots of bare bone strings, so a `frozen_string_literal: true` might come in handy